docs/src/whatsnew/latest.rst

```diff
@@ -52,6 +52,9 @@ This document explains the changes made to Iris for this release
    printing these objects skips metadata elements that are set to None or an
    empty string or dictionary. (:pull:`4040`)
 
+#. `@Badboy-16`_ implemented a ``CubeList.copy()`` method to return a
```

Comment:
This link also needs creating toward the bottom of the file - [currently L172-174](https://github.com/SciTools/iris/blame/956df9912a1bec53feecec0504a0383f0dab7ea5/docs/src/whatsnew/latest.rst#L172-L174).

That will fix the test failures.